### PR TITLE
Fix the issues in the gRPC and gRPC security BBEs

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Fix the issues in the gRPC and gRPC security BBEs.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/grpc-service-oauth2/grpc_service_oauth2.md [hunk 1]
```diff
@@ -1,18 +1,47 @@
 # Service - OAuth2
 
-A gRPC service/resource can be secured with OAuth2 and by enforcing authorization optionally. Then, it validates the OAuth2 token sent in the `Authorization` metadata against the provided configurations. This calls the configured introspection endpoint to validate.
-
-Ballerina uses the concept of scopes for authorization. A resource declared in a service can be bound to one/more scope(s). The scope can be included in the introspection response using a custom claim attribute. That custom
+A gRPC service/resource can be secured with OAuth2 and by enforcing
+authorization optionally. Then, it validates the OAuth2 token sent in the
+`Authorization` metadata against the provided configurations. This calls the
```
Comment (dilanSachi) [inline examples/grpc-service-oauth2/grpc_service_oauth2.md:5]: When we break these sentences into newlines, the website shows them in newlines even though the md file doesnt show it. So we have to keep them in the same line.

File: examples/grpc-service-oauth2/grpc_service_oauth2.md [hunk 2]
```diff
@@ -1,18 +1,47 @@
...
+configured introspection endpoint to validate.
+
+Ballerina uses the concept of scopes for authorization. A resource declared
+in a service can be bound to one/more scope(s). The scope can be included
+in the introspection response using a custom claim attribute. That custom
 claim attribute also can be configured as the `scopeKey`.
 
-In the authorization phase, the scopes of the service/resource are compared against the scope included in the introspection response for at least one match between the two sets.
+In the authorization phase, the scopes of the service/resource are compared
+against the scope included in the introspection response for at least one
```
Comment (dilanSachi) [inline examples/grpc-service-oauth2/grpc_service_oauth2.md:14]: same case

File: examples/grpc-service-ssl-tls/grpc_service_ssl_tls.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # Service - SSL/TLS
 
-You can use the gRPC listener to connect to or interact with a gRPC client secured with SSL/TLS. Provide the `grpc:ListenerSecureSocket` configurations to the server to expose an HTTPS connection over HTTP/2.
-
-For more information on the underlying module, see the [gRPC module](https://lib.ballerina.io/ballerina/grpc/latest/).
+You can use the gRPC listener to connect to or interact with a gRPC client secured with SSL/TLS.
+Provide the `grpc:ListenerSecureSocket` configurations to the server to
+expose an HTTPS connection over HTTP/2.
```
Comment (dilanSachi) [inline examples/grpc-service-ssl-tls/grpc_service_ssl_tls.md:5]: same case

File: examples/grpc-client-ssl-tls/grpc_client_ssl_tls.md [hunk 4]
```diff
@@ -1,10 +1,8 @@
 # Client - SSL/TLS
 
 You can use the gRPC client to connect or interact with a gRPC listener secured with SSL/TLS.
-
-Provide the `grpc:ClientSecureSocket` configurations to the client to initiate an HTTPS connection over HTTP/2.
-
-For more information on the underlying module, see the [gRPC module](https://lib.ballerina.io/ballerina/grpc/latest/).
+Provide the `grpc:ClientSecureSocket` configurations to the client to
+initiate an HTTPS connection over HTTP/2.
```
Comment (dilanSachi) [inline examples/grpc-client-ssl-tls/grpc_client_ssl_tls.md:5]: same case
Check other places also

**Review by dilanSachi — APPROVED** [review]

File: examples/grpc-client-ssl-tls/grpc_client_ssl_tls.md (hunk 4 above)
Comment (praneesha) [reply examples/grpc-client-ssl-tls/grpc_client_ssl_tls.md]: Fixed via https://github.com/ballerina-platform/ballerina-distribution/pull/3122/commits/f59075869e675ac29a5b3aae1f1828eee43ee66f.

File: examples/grpc-service-ssl-tls/grpc_service_ssl_tls.md (hunk 3 above)
Comment (praneesha) [reply examples/grpc-service-ssl-tls/grpc_service_ssl_tls.md]: Fixed via https://github.com/ballerina-platform/ballerina-distribution/pull/3122/commits/f59075869e675ac29a5b3aae1f1828eee43ee66f.

File: examples/grpc-service-oauth2/grpc_service_oauth2.md (hunk 2 above)
Comment (praneesha) [reply examples/grpc-service-oauth2/grpc_service_oauth2.md]: Fixed via https://github.com/ballerina-platform/ballerina-distribution/pull/3122/commits/f59075869e675ac29a5b3aae1f1828eee43ee66f.

File: examples/grpc-service-oauth2/grpc_service_oauth2.md (hunk 1 above)
Comment (praneesha) [reply examples/grpc-service-oauth2/grpc_service_oauth2.md]: Fixed via https://github.com/ballerina-platform/ballerina-distribution/pull/3122/commits/f59075869e675ac29a5b3aae1f1828eee43ee66f.

**Pull request merged**